app/command_loadtest.go

```diff
@@ -339,7 +347,7 @@ func (me *LoadTestProvider) ThreadedPostCommand(a *App, args *model.CommandArgs,
 	}
 
 	client := model.NewAPIv4Client(args.SiteURL)
-	client.MockSession(args.Session.Token)
+	//client.MockSession(args.Session.Token)
```

Comment:
Is this commented line needed?

Reply:
Woopsie. Yes, a replacement for that is needed.